### PR TITLE
Add pages for latexmk and pdflatex

### PR DESCRIPTION
**Pull request by 85pando**
Pdflatex and latexmk are ywo tools to create Output documents from LaTeX source files.

### CODE REVIEW

File: pages/common/latexmk.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# latexmk
+
+> Compile LaTeX source files into finished documents. Does multiple runs when
+> the output will change.
```
Comment (waldyrious) [inline pages/common/latexmk.md]: 1) Put each sentence into its own line
2) The second sentence isn't clear. Can you rephrase it?

File: pages/common/latexmk.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Compile a dvi document.
```
Comment (waldyrious) [inline pages/common/latexmk.md]: Spell out what "dvi" stands for, in parenthesis

File: pages/common/latexmk.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`latexmk {{source.tex}}`
+
+- Compile a pdf document.
+
+`latexmk -pdf {{source.tex}}`
+
+- Clean up temporary tex files created for a single tex file.
```
Comment (waldyrious) [inline pages/common/latexmk.md]: suggestion: single --> specific

**Comment by 85pando** [comment]
* Tried to rephrase the sentence about multiple runs.
* Elongated dvi in parentheses.
* reordered commands.

    * Should I put the `-C` flag also before? I ordered it like this, because in my experience it's not that often that you want to delete your outputfiles.

File: pages/common/pdflatex.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# pdflatex
+
+> Compile pdf document from LaTeX source files.
+
+- Compile pdf document:
```
Comment (waldyrious) [inline pages/common/pdflatex.md]: "Compile *a* pdf document" (as in the other command)

File: pages/common/pdflatex.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+`pdflatex {{source.tex}}`
+
+- Compile pdf document, halt on each error:
```
Comment (waldyrious) [inline pages/common/pdflatex.md]: "Compile **a** pdf document, halt**ing** on each error"

**Comment by waldyrious** [comment]
Thanks for the improvements, much better now :+1: 

> Should I put the -C flag also before?

No, I think you made a good call :)

Before I can endorse this, could you address the inline comments I left? Also the one about "single" --> "specific", if it was an overlook -- otherwise please explain why you disagree.

**Comment by 85pando** [comment]
* Changed the descriptions of the pdf version as suggested.
* Added an additional command with no options (did not know it also does this ☻ )

For the "single" --> "specific" comment, I understood it as a comment on my ordering of the commands of `latexmk -c {{source.tex}}` coming before `latexmk -c` which I then changed.
If adding the command without options was what you meant, all is fine now, otherwise please advise.

**Comment by waldyrious** [comment]
Sorry if I wasn't clear. I meant changing the description from "Clean up temporary tex files created for a **single** tex file:" to "Clean up temporary tex files created for a **specific** tex file:" (although "single" works, I believe "specific" gets the idea across better)

**Comment by 85pando** [comment]
Allgright, got you now ☺

This should be it then.

**Comment by waldyrious** [comment]
Looks great! :+1:

**Comment by rubenvereecken** [comment]
Live and learn, thanks a lot @85pand, I could actually use this one :+1:

**Comment by 85pando** [comment]
Isn't that the essence of tldr? ☻
